openedx/core/djangoapps/auth_exchange/views.py

```diff
@@ -128,21 +128,45 @@ def _get_path_of_arbitrary_backend_for_user(user):
                 return backend_path
 
     @staticmethod
-    def _is_grant_password(request):
+    def _raise_exception_if_grant_is_not_password(request):
         """
         Check if the access token provided is DOT based and has password type grant.
         """
```

Comment:
1. Nit: Possible function rename to be positive instead of negative.
2. Updated docstring
```suggestion
    def _ensure_access_token_has_password_grant(request):
        """
        Ensures the access token provided has password type grant.
        """
```